### PR TITLE
[Release 1/25/2024] NuGet.jobs Dev to main

### PR DESCRIPTION
**Pull request by lyndaidaii**

### CODE REVIEW

**Review by agr — APPROVED** [review]

File: src/ArchivePackages/ArchivePackages.csproj [hunk 1]
```diff
@@ -69,6 +69,8 @@
     </PackageReference>
     <PackageReference Include="Dapper.StrongName" />
     <PackageReference Include="Microsoft.Data.Services.Client" />
+    <PackageReference Include="System.Data.SqlClient">
+    </PackageReference>
```
Comment (erdembayar) [inline src/ArchivePackages/ArchivePackages.csproj:73]: Remove this one

Comment (erdembayar) [reply src/ArchivePackages/ArchivePackages.csproj:73]: Sorry, it's already merged code. Resolving.

File: src/ArchivePackages/ArchivePackages.csproj [hunk 2]
```diff
@@ -69,6 +69,8 @@
     </PackageReference>
     <PackageReference Include="Dapper.StrongName" />
     <PackageReference Include="Microsoft.Data.Services.Client" />
+    <PackageReference Include="System.Data.SqlClient">
```
[the remark this answers is not in the document]
Comment (erdembayar) [reply src/ArchivePackages/ArchivePackages.csproj:72]: Sorry, it's already merged code. Resolving.

[the remark this answers is not in the document]
Comment (lyndaidaii) [reply src/ArchivePackages/ArchivePackages.csproj:72]: Interesting, it didn't complain during build. I will open another PR to fix it

**Pull request merged**